shopfloor_mobile_base/static/wms/src/main.js

```diff
@@ -92,6 +92,8 @@ new Vue({
         });
         event_hub.$on("profile:selected", function (profile) {
             self.profile = profile;
+            const odoo = self.getOdoo({usage: "profile"});
```

Comment:
We shouldn't do any call if not required IMO.
You could check for a flag on `shopfloor_app_info`.